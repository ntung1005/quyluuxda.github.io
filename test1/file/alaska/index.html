<!DOCTYPE html>
<html>
	<head>
		<title>Alaska</title>
		<link rel="stylesheet" type="text/css" href="style.css">
		<link rel="icon" type="image/png" sizes="32x32" href="../../anh.png">
		<link rel="icon" type="image/png" sizes="16x16" href="../../anh.png">
	</head>
	<body>
		<div class="title">
			<div>
				<h1>Alaska</h1>
			</div>
		</div>	
		<div class="grid-container">
  			<div class="grid-item"><a href="img/alaska1.jpg"><img alt="Alaska" src="img/alaska1.jpg"></a></div>
  			<div class="grid-item"><a href="img/alaska2.jpg"><img alt="Alaska" src="img/alaska2.jpg"></a></div>
  			<div class="grid-item"><a href="img/alaska3.jpg"><img alt="Alaska" src="img/alaska3.jpg"></a></div>	
 		</div>	
 		<div class="content">
 			<h3>Nguồn gốc</h3>
 			<p>Alaska hay chính là là Alaska Malamute hoặc Mahlemuts là một trong những loài chó kéo xe tại vịnh Kotzebue, vùng đất Alaska. Những người dân nơi đây đã nuôi dưỡng và huấn luyện chúng để làm việc kéo xe dưới thời tiết lạnh giá. Sau đó, khi vùng đất này trở thành 1 bang của nước Mỹ thì Alaska nghiễm nhiên trở thành một loài động vật của Mỹ.</p>
 			<h3>Đặc điểm hình dáng</h3>
 			<p>Trong những bộ phim nước ngoài quay cảnh chốn rừng núi Bắc Cực, chắc chắn bạn đã bắt gặp hình ảnh những chú chó sói. Alaska khi nhìn qua cũng giống những chú chó sói ấy nhưng khi quan sát kỹ hơn, bạn sẽ phân biệt được.<br>

			Alaska được hiệp hội chó Hoa Kỳ AKC phân thành 3 loại theo kích thước lớn dần là Standard, Large Standard và Giant.<br>

			Chiều cao trung bình của chúng là từ 63,5 cm đến 68,5cm. Riêng dòng Giant có thể cao tới 70cm và cân nặng có thể lên đến 75kg. Sở dĩ Alaska được chọn làm chó kéo xe vì chân của chúng rất lớn và cơ bắp, bộ khung xương cao to, đặc biệt xương chân và các khớp xương chân rất phát triển.<br>

			Bộ lông của chúng cực dày, mềm và màu sắc biến thiên dần từ bụng tới sống lưng và mặt. Nếu như phần bụng có màu trắng toát thì sống lưng sẽ đậm sang màu khác và khuôn mặt thì phân thành các mảng trắng.<br>

			Cũng như giống Husky, để thích nghi với cái rét lạnh giá Bắc Cực, Alaska có 2 lớp lông là lớp lông dài bên ngoài không thấm nước và lớp lông ngắn bên trong mềm và bông như lông cừu. Cặp mắt của chúng quy định theo màu lông và thường là mắt màu nâu, màu đen, màu xanh… Mắt chúng hình quả hạnh nhân và đặc biệt có đôi tai đầy lông tơ.<br>
			Nói về bộ lông có lẽ Alaska là một trong những loài chó sở hữu bộ lông đẹp nhất, nhất là phần lông đuôi dài và xõa nhưng cũng chính vì vậy mà người nuôi phải đầu tư nhiều thời gian để chăm sóc lông cho chúng.<br>

			Về màu lông, chúng cũng có màu lông đa dạng tương tự Husky nhưng chủ yếu là màu xám trắng, xám lông chồn phối với trắng, đen trắng và thậm chí có những chú trắng toát toàn thân. Tuy màu lông khác nhau nhưng phần lông 4 chân và mõm luôn luôn là màu trắng.<br>

			Khi di chuyển Alaka luôn thẳng người, ngẩng cao đầu, mắt mở to tròn và luôn trong tư thế quan sát và tìm hiểu xung quanh.</p>
 			<h3>Đặc điểm tính cách</h3>
 			<p>Alaska được đánh giá là một trong những giống chó thông minh nhất. Tính cách của chúng như đã nói, rất hiền lành, nghe lời chủ, yêu thích trẻ nhỏ và luôn hòa thuận với các vật nuôi khác. Vốn dĩ thông minh nên Alaka luôn trong tư thế tò mò khám phá và rất hiếu động, chúng có thể tự đi rất xa mà vẫn tìm được đường về.<br>
 			Trong gia đình chủ nuôi, do tập tính bầy đàn nên Alaska thường coi chủ là con đầu đàn và luôn phục tùng mọi mệnh lênh, sẵn sàng bảo vê chủ. Cũng chính vì tập tính bầy đàn nên bạn cần cho phép chúng làm những điều mà chính yêu thích như chạy nhảy ở những nơi rộng thoáng.<br>

			Hãy từ bỏ trò chơi ném bóng, ném đồ vật với loài pet này bởi chúng không có thói quen nhặt đồ, nhặt bóng đâu nhé.
 			</p>
 		</div>
	</body>
</html>