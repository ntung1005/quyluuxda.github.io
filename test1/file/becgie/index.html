<!DOCTYPE html>
<html>
	<head>
		<title>Becgie</title>
		<link rel="stylesheet" type="text/css" href="style.css">
		<link rel="icon" type="image/png" sizes="32x32" href="../../anh.png">
		<link rel="icon" type="image/png" sizes="16x16" href="../../anh.png">
	</head>
	<body>
		<div class="title">
			<div>
				<h1>Becgie</h1>
			</div>
		</div>	
		<div class="grid-container">
  			<div class="grid-item"><a href="img/becgie1.jpg"><img alt="Becgie" src="img/becgie1.jpg"></a>Becgie Đức</div>
  			<div class="grid-item"><a href="img/becgie2.jpg"><img alt="Becgie" src="img/becgie2.jpg"></a>Becgie Bỉ</div>
  			<div class="grid-item"><a href="img/becgie3.jpg"><img alt="Becgie" src="img/becgie3.jpg"></a>Becgie Nga</div>	
 		</div>	
 		<div class="content">
 			<h3>Nguồn gốc</h3>
 			<p>Chó Becgie hay còn gọi là chó chăn cừu, một giống chó có xuất xứ từ nước Đức. Với thân hình vạm vỡ, sức chạy khỏe và trí thông minh, trung thành tuyệt đối, tại Đức loài chó này được sử dụng trong các quân đội. Ban đầu Becgie chỉ có dòng duy nhất từ nước Đức nhưng đến nay đã phân hóa thành các dòng khác nhau, bao gồm: Becgie Đức, Becgie Nga và Becgie Bỉ.</p>
 			<h3>Đặc điểm của chó Becgie</h3>
 			<b>Becgie Đức</b>
 			<p>Đặc điểm nhận dạng dễ nhất của Becgie Đức là chúng có thân hình to lớn hơn các loại becgie khác trên thế giới.<br>
 			Becgie Đức đã được nhân giống khắp thế giới và đến nay ngoại hình của chúng khác xa với chú chó Hektor Linksrhein đầu thế kỷ XIX. Đầu của chúng to hơn, tai cũng dài và lớn hơn còn lưng thì cong xuống. Becgie Đức được rất nhiều người trên thế giới ưa chuộng và đặc biệt chúng được các lực lượng quân đội, cứu hộ, cảnh sát trên khắp thế giới chọn nuôi và huấn luyện.<br>

			Với chiếc mũi siêu thính, chó Becgie Đức được chọn thực hiện các vụ tìm kiếm cứu hộ và thậm chí để đánh hơi tội phạm, ma túy. Thân hình chúng cực vạm vỡ và dẻo dai nên được nhiều đơn vị huấn luyện để tham gia chiến đấu hoặc làm cảnh vệ, trấn áp tội phạm.<br></p>
			
			<b>Becgie Bỉ</b>
			<p>Becgie Bỉ được lai tạo vào những năm cuối thể kỷ XIX từ chó becgie Đức bản địa. Tuy vậy đến nay Becgie Bỉ vẫn chưa đươc AKC cùng nhiều tổ chức khác công nhận là một giống chó riêng mà vẫn là cùng họ với Becgie Đức.<br>

			Becgie Bỉ có ngoại hình hơi giống chó Becgie Đức thời kỳ đầu với cái đầu nhỏ, tai cũng nhỏ hẹp, mõm dài, lưng thẳng và chỉ hơi cong ở phần hông. Tuy khác về ngoại hình nhưng tính cách trung thành, sự mạnh mẽ và trí thông minh thì không hề thua kém. Cũng chính vì thế mà tại Bỉ và một vài nước châu Âu đã chọn Manilois để phục vụ trọng quân đội, cảnh sát. <br>
 			</p>
			<b>Becgie Nga</b>
			<p>Loài chó này xuất hiện ở Nga vào những năm 1930, là kết quả từ sự lai tạo giữa Becgie Đức và chó bản địa của Nga. Becgie Nga tiếp nối được sức mạnh và trí thông minh của Becgie Đức nhưng lại có khả năng chịu lạnh cực tốt để thích nghi với khí hậu nước Nga. Tại Nga và các nước Đông Âu, giống chó này được huấn luyện đển thực hiện nhiệm vụ tuần tra biên giới.<br>

			Becgie Nga ngoài bộ lông màu sáng hơn thì đặc điểm ngoại hình gần giống với Becgie Đức thời kỳ đầu với thân hình cơ bắp, chân dài, đầu nhỏ, lưng thẳng và tai dơi. So với Becgie Bỉ và Đức thì Becgie Nga ít phổ biến hơn trên thế giới, đa số tập trung ở Nga và một số nước Đông Âu.<br>
 			</p>
 		</div>
	</body>
</html>