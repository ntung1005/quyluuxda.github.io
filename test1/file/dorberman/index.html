<!DOCTYPE html>
<html>
	<head>
		<title>Dorberman</title>
		<link rel="stylesheet" type="text/css" href="style.css">
		<link rel="icon" type="image/png" sizes="32x32" href="../../anh.png">
		<link rel="icon" type="image/png" sizes="16x16" href="../../anh.png">
	</head>
	<body>
		<div class="title">
			<div>
				<h1>Dorberman</h1>
			</div>
		</div>	
		<div class="grid-container">
  			<div class="grid-item"><a href="img/dorberman1.jpg"><img alt="Dorberman" src="img/dorberman1.jpg"></a></div>
  			<div class="grid-item"><a href="img/dorberman2.jpg"><img alt="Dorberman" src="img/dorberman2.jpg"></a></div>
  			<div class="grid-item"><a href="img/dorberman3.jpg"><img alt="Dorberman" src="img/dorberman3.jpg"></a></div>	
 		</div>	
 		<div class="content">
 			<h3>Nguồn gốc</h3>
 			<p>Dobermann được ra đời năm khoảng năm 1890 tại Đức bởi Louis Dobermann. Ông ta làm nghề thu thuế và thường xuyên phải đi lại ở những vùng đất đầy rẫy nguy hiểm và trộm cướp. Với công việc như vậy, ông cần một giống có những phẩm chất như: thông minh, trung thành, gan dạ, thính nhạy, khỏe mạnh, có thể lực tốt, có khả năng bảo vệ và tấn công cao vì thế ông đã tìm cách gây giống và tạo nên Dobermann.<br>

			Mọi người tin rằng Dobermann là sự kết hợp của giống German Pinscher, Rottweiler, Thuringian Shepherd, Greyhound đen, Great Dane, Weimaraner, German Shorthaired Pointer, Beauceron và German Shepherd cổ. Tỷ lệ kết hợp như thế nào, cho đến nay, vẫn chưa chắc chắn, nhưng giới chuyên môn vẫn tin rằng ít nhất Dobermann có mang dòng máu của 4 trong số các loại kể trên.</p>
 			<h3>Đặc điểm hình dáng</h3>
 			<b>Màu sắc</b>
 			<p> 	Khi nhắc đến Dobermann, đa số mọi người nghĩ đến một con Dobermann màu đen với các viền nâu vàng. Thực tế, Dobermann có 2 gene màu khác nhau (giả sử gene A và B) tương tác để có thể tạo ra bốn loại kiểu hình về màu sắc khác nhau. Con Dobermann với màu truyền thống đen hoặc đen và vàng là do chúng mang allele trội của cả hai gene (A - B -). Một loại biến chủng thường thấy ở Dobermann có bộ lông màu đỏ hay đỏ và vàng là do chúng đồng hợp lặn về 1 gene (vd. A - bb). Đó là cách gọi ở Mỹ còn ở các nơi khác, những con Dobermann như vậy được gọi là màu nâu vì bộ lông phủ toàn màu nâu đỏ sẫm với những viền vàng. Tuy nhiên, đối với những con Dobermann đồng hợp lặn ở gene màu kia (aa B-) thì có màu lông xanh xám. Một con Dobermann có tổ hợp gene đồng hợp lặn ở cả hai gene (aa bb), rất hiếm gặp trong tự nhiên, thì có màu vàng sáng và được gọi với cái tên là "isabella".<br>

			Trong thập niên 1970, một con Dobermann có màu thứ năm, một màu trắng hoàn toàn đã được sinh ra và từ đó di truyền màu lông này cho các thế hệ con cháu của nó. Quá trình nội phối nghiêm ngặt này diễn ra trong một vài thế hệ, theo phương thức mà các nhà chọn giống gọi là "cố định" đột biến, sau đó đã được tung ra thị trường. Những con Dobermann có màu lông như vậy đã mang một đột biến di truyền mới, tác nhân ngăn cản quá trình tổng hợp sắc tố, bất kể nó mang kiểu gene gì trong 2 gene màu nói trên (gene A và B). Thực ra, đó là những con chó mắc bệnh bạch tạng.<br>

			Mặc dù nhiều chủ nhân của những con chó Dobermann trắng thì thấy màu lông trắng khá đẹp, nhưng cũng như các con vật bị bệnh bạch tạng khác, chúng mang nguy cơ tiềm ẩn mắc bệnh ung thư và một số bệnh khác. Những con bạch tạng này cần tránh tối đa tiếp xúc với ánh sáng mặt trời. Vì nguy cơ mắc bệnh và chết sớm của những con Dobermann trắng này hầu như là chắc chắn nên ngày càng có nhiều người kêu gọi ngừng ngay việc chọn giống và thương mại những con bệnh bạch tạng, đó là một hành vi đối xử độc ác của con người đối với vật nuôi. Một vài quốc gia đã ban hành luật cấm việc chọn giống nhằm mục đích tạo ra những con Dobermann trắng, nhưng cho phép những nhà chọn giống có thể nuôi dưỡng những con Dobermann này nếu không để việc nội phối tạo ra các thế hệ bạch tạng tiếp theo.<br>		</p>
			<b>Đuôi</b>
 			<p> 	Nhìều người ngạc nhiên khi thấy những con Dobermann với đuôi rất ngắn. Thực tế đuôi của Dobermann thường dài hơn những giống chó khác. Nhưng thông thường, đuôi của Dobermann được cắt vài ngày sau khi được sinh ra. Lý do của việc này là Dobermann với đuôi ngắn (gần như cụt) là một điển hình về Doberman, đặc điểm này gắn liền với Doberman, do người tạo ra giống chó này là ông Louis Dobermann đã mường tượng ra chúng như vậy. Đặc điểm này còn làm cho Dobermann trông ngầu và dữ dằn hơn.<br>	

			Lý do khác là vấn đề công việc khi Dobermann làm nhiệm vụ của chó cảnh sát là bắt cướp. Kể tấn công có thể dễ dàng nắm đuôi Dobermann và điều đó trở thành một điểm yếu. Còn lý do khác nữa là đuôi Dobermann dễ bị gãy nếu để bình thường, một khi đã gãy thì rất khó chữa và sẽ gây đau đớn cho Doberman. Tuy vậy, vẫn có một số ít người muốn Dobermann với cái đuôi nguyên vẹn. Thường thì họ phải đặt vấn đề này với người bán sớm vì chuyện cắt đuôi Dobermann phải được tiến hành sớm sau khi vừa sinh ra.<br>			</p>
 			<b>Tai</b>
 			<p> 	Chuyện Dobermann cần phải được cắt tai để có một đôi tai dựng đứng trong khoảng 7 đến 9 tuần đầu từ lúc mới sinh là không chính xác, chuyện này có thể để đến 6 tháng hoặc 1 năm cũng được. Nhiều người tỏ ra không thích chuyện cắt tai Dobermann vì họ nghĩ rằng điều đó làm Dobermann rất đau đớn. Từ trước đến nay vẫn chưa ai đem Dobermann được cắt tai và Dobermann để tai tự nhiên ra so sánh, nhưng người ta vẫn tin rằng Dobermann được cắt tai có thể tránh được những vấn đề về tai như nhiễm trùng cao hơn.		</p>
 			<b>Thể chất</b>
 			<p> 	Dobermann là giống chó có sức khỏe rất tốt, ít bệnh tật, tuổi thọ trung bình 11-13 năm. Tuy nhiên khả năng chống chịu thời tiết khắc nghiệt của chúng không được tốt lắm. Ở xứ lạnh vào những hôm nhiệt độ ngoài trời quá thấp có băng tuyết thì không nên để Dobermann ở ngoài trời (vì lông của chúng ngắn, chịu lạnh không tốt bằng nhưng giống lông xù).		</p>
 			<h3>Đặc điểm tính cách</h3>
 			<p>Được lai tạo hàng thế kỷ để trở thành loài chó canh gác, Doberman có được các phẩm chất lý tưởng cho công việc này: Nhậy cảm, nhiệt huyết, có sức mạnh và sự bền bỉ phi thường. Hơn thế nữa, đây là một trong những giống chó vô cùng thông minh và dễ dạy bảo. Quyết đoán, không hề biết sợ hãi nhưng không hoang dã, trung thành, tận tuỵ và tình cảm với gia đình chủ, chó Doberman gần như trở thành một thành viên chính thức của gia đình.Vì Doberman rất thông minh, dễ dạy bảo, có khả năng tấn công, bảo vệ rất tốt nên trong thực tế chúng thường được huấn luyện để phục vụ nhiều mục đích khác nhau (trong đó có cả mục đích xấu).<br>

			Phim ảnh đã từng có thời dựng rất nhiều về Doberman như Doberman Gang (1972), Daring Dobermans(1973), The Amazing Dobermans(1976), Eyes of an Angel(1991)....,trong các bộ phim này đôi lúc Doberman xuất hiện với hình ảnh dữ dằn (Doberman Gang) nên nhiều người không hiểu biết về giống này nghĩ rằng giống này hung dữ nhưng thực ra Dobermann rất đáng yêu và thông minh (xem phim Eyes of an Angel), chúng rất hiếm khi tự ý tấn công con người, trừ trường hợp nó cảm thấy có điều gì nguy hiểm đang xảy ra với chủ hoặc gia đình nó hoặc khi nó được huấn luyện và ra lệnh tấn công con người. Trong các cuộc thi về vâng lời, Doberman luôn được đánh giá là giống chó thông minh và vâng lời bậc nhất.<br></p>
 		</div>
	</body>
</html>