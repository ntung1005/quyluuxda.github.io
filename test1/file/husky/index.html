<!DOCTYPE html>
<html>
	<head>
		<title>Husky</title>
		<link rel="stylesheet" type="text/css" href="style.css">
		<link rel="icon" type="image/png" sizes="32x32" href="../../anh.png">
		<link rel="icon" type="image/png" sizes="16x16" href="../../anh.png">
	</head>
	<body>
		<div class="title">
			<div>
				<h1>Husky</h1>
			</div>
		</div>	
		<div class="grid-container">
  			<div class="grid-item"><a href="img/husky1.jpg"><img alt="Husky" src="img/husky1.jpg"></a></div>
  			<div class="grid-item"><a href="img/husky2.jpg"><img alt="Husky" src="img/husky2.jpg"></a></div>
  			<div class="grid-item"><a href="img/husky3.jpg"><img alt="Husky" src="img/husky3.jpg"></a></div>	
 		</div>	
 		<div class="content">
 			<h3>Nguồn gốc</h3>
 			<p>Chắc chắn khi lần đầu tiên thấy Husky, bạn sẽ liên tưởng đến những bộ phim nổi tiếng của nước ngoài ở các vùng đất lạnh giá, vậy chính xác xuất xứ của chúng từ đâu?<br>
 			Theo các tài liệu nghiên cứu, Husky xuất xứ từ vùng Đông Bắc Siberi nước Nga lạnh giá. Đây là vùng đất đã sản sinh ra rất nhiều loài chó nổi tiếng và sau đó chúng được mang đến các vùng đất Alaska của Mỹ. Từ đây, giống chó này được nuôi để làm việc, kéo xe bởi chúng có khả năng chạy rất nhanh và khỏe. Tuy nhiên, sau đó người dân ở đây đã không nuôi husky để nhằm mục đích đó nữa mà chuyển sang nuôi như một vật nuôi trong nhà. Đến nay, Husky vẫn được xếp vào hạng những vật nuôi đặc trưng của vùng đất Siberi.
 			</p>
 			<h3>Đặc điểm hình dáng</h3>
 			<p>Là một trong những dòng chó kéo xe cùng với Alaska, Sammoyed, Husky có bộ lông rất dày và dài. Nhiều người không am hiểu thường hay nhầm lẫn giữa Husky và Alaska vì hình dáng nhìn qua trông khá giống nhau. Tuy nhiên khi quan sát kỹ hơn sẽ thấy Husky có hình thể nhỏ hơn. Cụ thể con đực cao từ 53 – 58cm, nặng từ 20 – 27kg, còn con cái cao từ 51 – 56cm, nặng từ 16 – 23kg. Mặt husky trông dữ dằn hơn Alaska và lông cũng ngắn hơn nên vì vậy alaska trông to gấp đôi thậm chí gấp 3 lần husky.<br>

			Husky có rất nhiều màu và thường gặp nhiều nhất là màu đen trắng, nâu đỏ, xám, hồng phấn, màu trắng và màu “agouti”. Tuy có nhiều màu nhưng đa phần vùng lông ở các bộ phận như chân, mõm, đốm cuối đuôi thường là màu trắng.<br>

			Mắt husky có hình quả hạnh nhân hơi xếch lên và đặt cách nhau vừa phải. Nhìn vào đôi mắt của chúng toát lên vẻ hoang dã, lạnh lùng như những chú sói hoang.
			Màu mắt của husky rất đa dạng, điển hình thường gặp nhất là màu nâu, màu xanh dương, màu hổ phách, xanh lục, xanh lục nhạt, nâu lục… Thậm chí nhiều con lại có 2 mắt 2 màu hay có nhiều con mắt pha trộn nửa xanh nửa nâu trông rất đặc biệt.<br>

			Lông husky cấu thành từ 2 lớp rất dày bởi nguồn gốc của chúng từ xứ lạnh khắc nghiệt. Lớp lông bên trong xoăn nhẹ, dày và mềm như bông, còn lớp lông bên ngoài dài hơn, cứng hơn và thẳng. Lông bên ngoài có thể có 2 màu nên khi nhìn vào chúng có thể thấy 1 vùng lông đổi màu hết trắng sang đen rồi lại trắng. Mỗi năm, lông bên trong của husky sẽ bị rụng 2 lần và thay lông mới.<br>

			Màu mũi của husky quy định theo màu lông. Cụ thể lông màu xám thì mũi màu đen, lông màu đen thì mũi màu nâu, lông màu nâu thì mũi màu đỏ thẫm, lông màu trắng thì mũi màu xám nhạt.
			Mũi husky cũng luôn ướt và mát nhưng điều đặc biệt ở đây là những chiếc mùi xinh xắn đó có thể chuyển màu theo thời gian. Đa phần về mùa đông thiếu ánh sáng, mũi của chúng sẽ có màu nâu hoặc hồng, còn về mùa hè lại chuyển về màu ban đầu.<br>
			Chiếc đuôi của chúng được phủ lông rất dày, phần chỏm màu trắng và hơi cong tạo nên đốm bông trông rất đáng yêu. Khi di chuyển, husky sẽ thả thõng đuôi xuống.<br>

			Tai husky lúc nào cũng vểnh lên, hướng về phía trước và có hình tam giác. Sờ vào phần lông ở tai cảm thấy rất mềm, mượt và êm như nhung.<br>
 			<h3>Đặc điểm tính cách</h3>
 			<p>Không giống với hình ảnh chú chó sói gian ác, hung dữ như trong phim đâu nhé, husky có tính cách trái ngược hoàn toàn với vẻ ngoài của chúng. Có thể khi nhìn thấy bạn sẽ sợ chúng nhưng rồi sẽ rất bất ngờ khi tính cách của chúng cực hiền lành và thân thiện với con người.<br>

			Husky rất nghe lời chủ và khá là thông minh. Các gia đình nuôi Husky thường coi chúng như thành viên trong nhà bởi chúng sống rất tình cảm với chủ nhân và đặc biệt cực yêu thích trẻ em. Vốn dĩ có sức chạy rất khỏe nên husky rất thích được ra ngoài chơi và chạy nhảy nên mỗi ngày người nuôi phải dành cho chúng khoảng 2 tiếng đi dao và tập thể dục để chúng thích thú và cảm nhận được sự quan tâm.<br>

			Husky được gọi là “thánh biểu cảm” bởi khuôn mặt của chúng có thể biểu cảm được mọi cảm xúc một cách rất “deep”, từ vui buồn, giận dỗi, ngơ ngác, tội lỗi cho đến nịnh nọt…<br>
 			</p>
 		</div>
	</body>
</html>