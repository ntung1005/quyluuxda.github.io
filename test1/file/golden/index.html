<!DOCTYPE html>
<html>
	<head>
		<title>Golden</title>
		<link rel="stylesheet" type="text/css" href="style.css">
		<link rel="icon" type="image/png" sizes="32x32" href="../../anh.png">
		<link rel="icon" type="image/png" sizes="16x16" href="../../anh.png">
	</head>
	<body>
		<div class="title">
			<div>
				<h1>Golden</h1>
			</div>
		</div>	
		<div class="grid-container">
  			<div class="grid-item"><a href="img/golden1.jpg"><img alt="Golden" src="img/golden1.jpg"></a></div>
  			<div class="grid-item"><a href="img/golden2.jpg"><img alt="Golden" src="img/golden2.jpg"></a></div>
  			<div class="grid-item"><a href="img/golden3.jpg"><img alt="Golden" src="img/golden3.jpg"></a></div>	
 		</div>	
 		<div class="content">
 			<h3>Nguồn gốc</h3>
 			<p>Chó golden có tên chính xác là Golden Retriever, là một trong những giống chó năng động, ham chơi nhưng cũng rất thông minh, trung thành với chủ. Nhiều nơi nuôi golden để săn mồi, tha mồi, thậm chí là dò tìm ma túy.

			Golden xuất hiện đầu tiên tại các hòn đảo nước Anh, là kết quả của sự lai tạo giữa các giống Retriver lông thẳng màu vàng với Tweed Water Spaniels lông ngắn cùng các giống setters, spaniels, bloodhound và newfoundland. Tên gọi đầu tiên của Golden ngày xưa là Golden Flat – Coat.</p>
 			<h3>Đặc điểm hình dáng</h3>
 			<p>Golden sở hữu một ngoại hình gọn gàng, cân đối và khỏe khoắn. Nhìn tổng thể, ngoại hình của chúng khá đẹp mắt, thu hút mọi người. Kích thước chuẩn của một chú chó Golden Retriever trưởng thành là:<br>

			Con cái: cao 55-57 cm, nặng 27-32 kg.<br>
			Con đực: cao 58-61 cm, nặng 29-34 kg.<br>
			Chúng có một cái đầu khá to. Chiếc mũi màu đen cực thính giúp Golden có thể lần mò được mọi dấu vết. Đôi mắt màu nâu đen, to tròn thể hiện sự tinh nhanh. Mõm khá to cùng với bộ răng hàm chắc khỏe, sắc bén.<br>
			Hai tai của chó Golden khá dài, thường cụp xuống che mất ½ má. Cổ to, ngực rộng, cơ bắp phát triển tốt. Đuôi của chó Golden dài đến khuỷu chân, không bao giờ cong.<br>

			Giống chó Golden Retriver sở hữu một bộ lông dày và mềm mượt, có màu vàng kem. Bộ lông ấy chia thành 2 lớp chính. Lớp lông bên ngoài dài và hơi cứng. Lớp lông bên trong ngắn và mềm mượt hơn rất nhiều.<br>
 			</p>
 			<h3>Đặc điểm tính cách</h3>
 			Giống chó này rất thích làm hài lòng chủ nhân của nó. Đó là lý do tại sao chúng luôn sẵn sàng làm rất nhiều công việc nhà. Sự nỗ lực và tình yêu con người đã giúp cho chó Golden trở thành những người bạn không thể thiếu trong mỗi gia đình.
 			Golden Retriever cực kỳ thông minh. Chúng luôn biết điều chỉnh hành vi của mình sao cho phù hợp nhất các hoàn cảnh khác nhau. Không như những chú chó săn mồi khác, thường cứng đầu và không kiềm chế được sự cáu giận.<br>

			Trung thành, ngoan ngoãn và hài hước cũng là một trong những nét tính cách nổi bật của chó Golden. Chúng luôn mang đến niềm vui cho mọi người xung quanh. Nếu bạn đang buồn, chúng sẽ thấu hiểu điều đó và làm nhiều trò chọc ghẹo bạn cười.<br>

			Chó Golden Retriever rất năng động, hoạt bát, yêu thích các hoạt động ngoài trời. Dù là giống chó săn mồi nhưng Golden luôn thân thiện với con người và các vật nuôi khác. Sẽ chẳng bao giờ chúng làm hại đến ai nếu như không ai xâm phạm đến chúng.<br>

			Golden cũng sẽ làm tốt nhiệm vụ trông giữ nhà cửa nếu được giao. Khi có người lạ, chúng sẽ cất tiếng sủa để báo hiệu cho chủ nhân. Nếu nuôi Golden Retriever, bạn cần phải cho chúng sống trong tình cảm yêu thương, quan tâm chân thành. Bởi đó là những điều mà Golden mong muốn được nhận lại từ con người.<br>
 		</div>
	</body>
</html>